Snakefile

```diff
@@ -138,6 +138,25 @@ rule remove_outliers:
         pp.clean.remove_outliers(*input, *output)
 
 
+rule drop_na_rows:
+    input:
+        "outputs/{scenario}/{pipeline}.parquet",
+    output:
+        "outputs/{scenario}/{pipeline}_dropna.parquet",
+    benchmark:
+        "benchmarks/{scenario}/{pipeline}_dropna.txt"
+    params:
+        na_threshold=config.get("na_threshold", 0.0),
```

Comment:
@jfredinh - I assume you'd want to make the default 10% (because that's what you have in your PR?)